src/markdown-pages/install-with-kurl/setup-tls-certs.md

```diff
@@ -43,3 +43,32 @@ After the pod has been restarted direct your browser to `http://<ip>:8800` and r
     
 It's best to complete this process as soon as possible to avoid anyone from nefariously uploading TLS certificates.  After this process has completed, the vulnerability will be closed, and uploading new TLS certificates will be disallowed again.  In order to upload new TLS certificates you must repeat the steps above. 
 <br><br><br>
+
+## Registry
+
+The TLS certificate for the [registry add-on](/docs/add-ons/registry) will be renewed automatically at thirty days prior to expiration if the [ekco add-on](/docs/add-ons/ekco) is enabled with version 0.5.0+.
+
+To manually renew the certificate for the registry prior to automatic renewal, re-run the kURL install script.
+
+## Kubernetes Control Plane
+
+The certificates for the Kubernetes control plane will be renewed automatically at thirty days prior to expiration if the [ekco add-on](/docs/add-ons/ekco) is enabled with version 0.5.0+.
+
+To check how long certificates have remaining until expiration, run this command on every primary node:
+```bash
+kubeadm alpha certs check-expiration
+```
+
+Renewing the Kubernetes control plane certificates will trigger a restart of the Kubernetes API server, which may affect applications running in the cluster.
+
+If running an HA cluster with a load balancer, use this procedure to manually rotate the certificates at any point prior to the automatic rotatation deadline.
+
+1. Remove the first primary node from your load balancer's target group.
+1. Run `kubeadm alpha certs renew all` on the node.
```

Comment:
Is it possible to just trigger ekco to do this, since it knows how to do all those steps?

Reply:
It's not implemented where you can run these steps on demand for a single node.

Reply:
Well, I'd rather not run it for a single node at all, but trigger something that runs it for all nodes. Anyway, this is fine for now, but something to consider, if it comes up, is to help automate that a bit better, yet still allow folks to pick the timing of the update.